docs/helm/server.md

```diff
@@ -134,3 +134,131 @@ permissionScopeMapping:
 ```
  
 By default every user (including anonymous one) is member of group `everyone`.
+
+### Job resource defaults
+
+By default Armada-server will validate submitted jobs set some value for resource request and limit. 
+
+It also makes sure `resource request = resource limit`. 
+
+However if you remove the requirement for your users to set a resource request/limit or simply provide sensible defaults you can specify defaults with:
+
+```yaml
+scheduling:
+  defaultJobLimits:
+    memory: 8Gi
+    cpu: 1
+```
+
+The default will only be used when a submitted job doesn't specify a request/limit for the resource type specified.
+
+Using the above values, if a job was submitted with:
+
+```yaml
+  resources:
+    limits:
+      memory: 64Mi
+    requests:
+      memory: 64Mi
+```
+
+It will be transformed to this:
+
+```yaml
+  resources:
+    limits:
+      memory: 64Mi
+      cpu: 1
+    requests:
+      memory: 64Mi
+      cpu: 1
+```
+
+### Scheduling
+
+The default scheduling configuration can be seen below:
+
+```yaml
+scheduling:
+  queueLeaseBatchSize: 200
+  minimumResourceToSchedule:
+    memory: 100000000 # 100Mb
+    cpu: 0.25
+  maximalClusterFractionToSchedule:
+    memory: 0.25
+    cpu: 0.25
+```
+
+`queueLeaseBatchSize` This is the number of jobs at the top of a queue the scheduler will look at when trying to find one that'll fit in the available resource.
```

Comment:
I think this is just size of batch, if whole batch is scheduled, we will load additional `queueLeaseBatchSize` amount until we run out of time or global scheduling limit or the queue is empty.

Reply:
Never mind the effect is actually what is described.